### PR TITLE
Task #5236 - Admin create book pull 1

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/5236?issue_count=19&issue_position=11&next_issue_id=5234&prev_issue_id=5237

### CODE REVIEW

**Comment by taylq** [comment]
ready

File: app/controllers/admin/books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module Admin
+  class BooksController < BaseController
+    def index
+      @books = Book.select_attr.page(params[:page]).per(Settings.per_page).search params[:search]
+      respond_to do |format|
+        format.html {}
```
Comment (huuphuc93) [inline app/controllers/admin/books_controller.rb]: bỏ {}

File: app/controllers/admin/borrows_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class BorrowsController < BaseController
     before_action :find_borrow, :find_book, only: :update
 
     def index
-      @borrows = Borrow.select_attr.search params[:status], params[:search]
+      @borrows = Borrow.select_attr.page(params[:page]).per(Settings.per_page).search params[:status], params[:search]
```
Comment (huuphuc93) [inline app/controllers/admin/borrows_controller.rb]: Quá 80 ký tự xuống dòng đi em

File: app/views/admin/books/_book.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<tr>
+  <td class="row"><%= book_counter + 1 %></td>
+  <td><%= image_tag book.image.url ||= book.image.default_url %></td>
+  <td><%= link_to book.name, admin_book_path(book) %></td>
+  <td><%= book.category.name %></td>
```
Comment (huuphuc93) [inline app/views/admin/books/_book.html.erb]: dùng delegate

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**